.reuse/dep5

```diff
@@ -13,4 +13,16 @@ License: GPL-3.0-or-later
 
 Files: doc/img/*
 Copyright: 2019 The Atlite Authors
-License: CC-BY-4.0
\ No newline at end of file
+License: CC-BY-4.0
+
+Files: doc/workflow*
+Copyright: 2021 The Atlite Authors
+License: CC-BY-4.0
+
+Files: doc/examples/*.nblink
+Copyright: 2020-2021 The Atlite Authors
+License: CC-BY-4.0
+
+Files: .github/*
+Copyright: 2021 The Atlite Authors
+License: CC0-1.0
```

Comment:
`./reuse/dep5` is only intended for large directories or binary files (https://reuse.software/spec/#dep5).

Rather include the identifier into these files directly? For markdown that is
```
<!--
SPDX-FileCopyrightText: 2021 Jane

SPDX-License-Identifier: CC0-1.0
-->
```

or with the `reuse` tool
```
reuse addheader --copyright="2021 The Atlite Authors" --license="CC0-1.0" <file paths>
```

If you want I can do it.

Reply:
Ok, done. I have kept it for the images and `*nblink` which I think are automatically generated.

`reuse>=0.13` also supports `*.ipynb` files with `reuse addheader ...` which creates a `.license` file.

Reply:
The `*.nblink` files are JSONs manually created by us and used by the SPHINX extension `nbsphinx_link`. So you could also add the header to those.

Great to hear REUSE finally supports `*.ipynb`'s :)

Reply:
Although I think the `*.ipynb.license` structure is clear than adding the license to the `./reuse/dep5` file =/

Reply:
Do JSONs have comments? probably it would work out as `*.nblink.license`

Reply:
No, JSONs don't have comments. Will probably generate the `.license` file. So better leave it in `dep5` then :)